test/function_test.rb

```diff
@@ -35,7 +35,7 @@ def test_throwing_function
       context.call('f', 1)
     end
     assert_equal err.message, 'Error: foo bar'
-    assert_match(/1:23/, err.backtrace[0])
+    assert_match(/1:23/, err.backtrace[1]) unless RUBY_ENGINE == "truffleruby"
```

Comment:
Could you add an assert for truffleruby, even if it's off by 1 currently?

Reply:
Yes, assert added for TruffleRuby